examples/wdio/multiremote/mocha.test.js

```diff
@@ -21,7 +27,7 @@ describe('multiremote example', () => {
     })
 
     it('should read the message in browserB', () => {
-        const msgElemBrowserB = browserA.$('.inputMessage')
+        const msgElemBrowserB = browserB.$('.inputMessage')
```

Comment:
I believe this is an existing bug in the example, so I fixed it. I mainly use this file to manually verify my fix